### PR TITLE
Add T2T 2019 tournament wiki page

### PR DESCRIPTION
**Pull request by aMalteser**
-Adds TAG Tournament (T2T) to the tournament wiki page

### CODE REVIEW

**Comment by Joehuu** [comment]
Move `en.md` to `wiki/Tournaments/T2T/2019`.

Rename pull request title to something along the lines `Add T2T tournament wiki page`.

**Comment by aMalteser** [comment]
Thanks, I have added those changes 👍

File: wiki/Tournaments/TAG Tournament/T2T/2019/en.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+---
+tags:
+- TAG2 Tournament
+- TAG2Tournament
+- T2T
+- T 2 T
```
Comment (cl8n) [inline wiki/Tournaments/TAG Tournament/T2T/2019/en.md]: probably don't need this tag

File: wiki/Tournaments/TAG Tournament/T2T/2019/en.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+- [Registration form](https://docs.google.com/forms/d/e/1FAIpQLSc412ThxxCb10gzYy7v6uHox4vffuozopRs8NzwxUx8uVoUVQ/viewform?usp=sf_link)
+- [TAG Club Discord](https://discord.gg/VKt5kGa)
+- [TAG Club Twitter](https://twitter.com/TAGClub_osu)
+- [Challonge](https://challonge.com/T2T_2019)
+- **[Spreadsheet](https://docs.google.com/spreadsheets/d/13f9zVN8n1NjBqwK7C23W5iOW-7Y2d9np9Di1WnlUfjg/edit#gid=1150595067)**
+
+
+--------------------------------
+
+## Groups
```
Comment (cl8n) [inline wiki/Tournaments/TAG Tournament/T2T/2019/en.md]: can delete all of these sections up through Mappools, and add them back with later updates to this article

**Comment by Joehuu** [comment]
@aMalteser Can you merge this? https://github.com/aMalteser/osu-wiki/pull/3

**Comment by aMalteser** [comment]
@Joehuu No I can't, it is blocked as I need 1 approving review

File: wiki/Tournaments/T2T/2019/en.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+- [Registration form](https://docs.google.com/forms/d/e/1FAIpQLSc412ThxxCb10gzYy7v6uHox4vffuozopRs8NzwxUx8uVoUVQ/viewform?usp=sf_link)
+- [TAG Club Discord](https://discord.gg/VKt5kGa)
+- [TAG Club Twitter](https://twitter.com/TAGClub_osu)
+- [Challonge](https://challonge.com/T2T_2019)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/13f9zVN8n1NjBqwK7C23W5iOW-7Y2d9np9Di1WnlUfjg/edit#gid=1150595067)**
+
+## Mappools
+
+### Group Stage
+
```
Comment (Joehuu) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
```
Comment (Joehuu) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+### Qualifiers
```
Comment (Joehuu) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
### Qualifiers

```

File: wiki/Tournaments/T2T/2019/en.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+**[Download the mappack here! (32.7MB)](https://puu.sh/D4s4A/a995511d8a.zip)**
```
Comment (Joehuu) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
**[Download the mappack here! (32.7MB)](https://puu.sh/D4s4A/a995511d8a.zip)**

```

**Comment by aMalteser** [comment]
@Joehuu could you reapprove the changes, it automatically dimissed the review after I updated the mappooling team

File: wiki/Tournaments/T2T/2019/en.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+## Organisation
+
+The TAG2 Tournament 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| --: | :-- |
+| Hosts | ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178) |
+| Co-Hosts | ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Mappoolers | ![][flag_CA] [Legless](https://osu.ppy.sh/users/3224243), ![][flag_GB] [Doughy](https://osu.ppy.sh/users/5275937), ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178), ![][flag_US] [Junjou](https://osu.ppy.sh/users/7077648)|
```
Comment (Joehuu) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
| Mappoolers | ![][flag_CA] [Legless](https://osu.ppy.sh/users/3224243), ![][flag_GB] [Doughy](https://osu.ppy.sh/users/5275937), ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178), ![][flag_US] [Junjou](https://osu.ppy.sh/users/7077648) |
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
+# TAG2 Tournament 2019
+
+![T2T Logo](img/logo.png)
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
![T2T 2019 Logo](img/logo.png)
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+The **TAG2 Tournament 2019** (***T2T 2019***) is an international Tag Team VS. tournament hosted by [Mara](https://osu.ppy.sh/users/194294) and [Malteser](https://osu.ppy.sh/users/5218178). It is the 3rd installment of the TAG Tournament series. The tournament follows on from [Tayo](https://osu.ppy.sh/users/8910355)'s 2 previous TAG Tournaments [T4T 2017](https://osu.ppy.sh/community/forums/topics/624941) and [T4T 2018](https://osu.ppy.sh/community/forums/topics/789136).
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
The **TAG2 Tournament 2019** (***T2T 2019***) is an international Tag Team VS. tournament hosted by [Mara](https://osu.ppy.sh/users/194294) and [Malteser](https://osu.ppy.sh/users/5218178). It is the 3rd installment of the TAG Tournament series. The tournament follows on from [Tayo](https://osu.ppy.sh/users/8910355)'s 2 previous TAG Tournaments: [T4T 2017](https://osu.ppy.sh/community/forums/topics/624941) and [T4T 2018](https://osu.ppy.sh/community/forums/topics/789136).
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+| Quarterfinals | 2019-04-20/2019-04-21 |
+| Semifinals | 2019-04-27/2019-04-28 |
+| Finals | 2019-05-04/2019-05-05 |
+| Grand Finals | 2019-05-11/2019-05-12 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter, profile badge |
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, profile badge |
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter |
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter |
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter tag |
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 13]
```diff
@@ -0,0 +1,148 @@
...
+
+## Organisation
+
+The TAG2 Tournament 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
The TAG2 Tournament 2019 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 14]
```diff
@@ -0,0 +1,148 @@
...
+
+| Position | Member(s) |
+| --: | :-- |
+| Hosts | ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178) |
+| Co-Hosts | ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Mappoolers | ![][flag_CA] [Legless](https://osu.ppy.sh/users/3224243), ![][flag_GB] [Doughy](https://osu.ppy.sh/users/5275937), ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178), ![][flag_US] [Junjou](https://osu.ppy.sh/users/7077648) |
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
| Map Selectors | ![][flag_CA] [Legless](https://osu.ppy.sh/users/3224243), ![][flag_GB] [Doughy](https://osu.ppy.sh/users/5275937), ![][flag_FI] [Mara](https://osu.ppy.sh/users/194294), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178), ![][flag_US] [Junjou](https://osu.ppy.sh/users/7077648) |
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+| Referees | ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_BR] [Flow-](https://osu.ppy.sh/users/4222824), ![][flag_PL] [HAEN24](https://osu.ppy.sh/users/4390077), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_FI] [Laurakko](https://osu.ppy.sh/users/7253731), ![][flag_MY] [Stupid Idiot](https://osu.ppy.sh/users/8355574), ![][flag_US] [YoshiLover456](https://osu.ppy.sh/users/6843383), ![][flag_GR] [nikolomara](https://osu.ppy.sh/users/10077264), ![][flag_CH] [Icerite](https://osu.ppy.sh/users/7226287) |
+| Spreadsheets | ![][flag_TR] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_NO] [YokesPai](https://osu.ppy.sh/users/6399568) |
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
| Statisticians | ![][flag_TR] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_NO] [YokesPai](https://osu.ppy.sh/users/6399568) |
```

File: wiki/Tournaments/T2T/2019/en.md [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+- [TAG Club Discord](https://discord.gg/VKt5kGa)
+- [TAG Club Twitter](https://twitter.com/TAGClub_osu)
+- [Challonge](https://challonge.com/T2T_2019)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/13f9zVN8n1NjBqwK7C23W5iOW-7Y2d9np9Di1WnlUfjg/edit#gid=1150595067)**
+
+## Mappools
+
+### Qualifiers
+
+**[Download the mappack here! (32.7MB)](https://puu.sh/D4s4A/a995511d8a.zip)**
```
Comment (TPGPL) [inline wiki/Tournaments/T2T/2019/en.md]:
```suggestion
**[Download the mappack here! (33 MB)](https://puu.sh/D4s4A/a995511d8a.zip)**
```

**Pull request merged**